graylog2-server/src/main/java/org/graylog/failure/FailureSubmissionService.java

```diff
@@ -155,7 +172,7 @@ public void submitIndexingErrors(Collection<IndexingError> indexingErrors) {
                                 }
                             })
                             .map(this::fromIndexingError)
```

Comment:
.peek(ie -> updateIndexingFailureMetric(ie.message()))

Reply:
I pulled this out into a separate statement since `peek` is not guaranteed to be called on all stream elements.